fendermint/actors/Cargo.toml

```diff
@@ -1,38 +1,38 @@
 [package]
-name = "fendermint_actors"
-version = "0.1.0"
+name = "recall_fendermint_actors"
+version.workspace = true
```

Comment:
I moved all of the explicit version numbers to the root Cargo.toml so we don't have to try to sync them in many places.